framework/decode/vulkan_virtual_swapchain.cpp

```diff
@@ -411,6 +415,11 @@ VkResult VulkanVirtualSwapchain::QueuePresentKHR(PFN_vkQueuePresentKHR
     }
     queue = queue_info->handle;
 
+    if (performance_mode_)
+    {
+        return func(queue, present_info);
+    }
+
```

Comment:
Hi, thanks for another interesting PR from ARM.

I'm wondering what appears on the screen when this PR is in effect. Do you get any valid images (but not necessarily all frames or in the right order) out of it?

Reply:
There would be no visual output.